buildnotifylib/core/projects.py

```diff
@@ -88,7 +89,11 @@ def __init__(self, server_config, timeout, connection=HttpConnection()):
     def get_data(self):
         print("checking %s" % self.server_config.url)
         try:
-            data = self.connection.connect(self.server_config, self.timeout)
+            headers = {}
```

Comment:
I would have pushed the header addition to http_connection as well because we are already aware of server object inside it. However looking at it now after a lot of time. I'm not too sure if http_connection should even be server aware so we can leave this here.